config/default/testRunner.conf.php

```diff
@@ -369,6 +369,33 @@
             'action' => 'messages',
 
             /**
+             * List of actions which can be send to server using common communication channel
+             * @type array
+             */
+            'syncActions' => array(
+                /**
+                 * Action name
+                 * @type array
+                 */
+                'move' => array(
+                    /**
+                     * Whether sending of action request can be postponed (true) of should be send immediately (false).
+                     * @type string
+                     */
+                    'deferred' => false,
+                ),
+                'skip' => array(
+                    'deferred' => false,
+                ),
+                'storeTraceData' => array(
+                    'deferred' => true,
+                ),
+                'timeout' => array(
+                    'deferred' => false,
+                ),
+            ),
```

Comment:
I keep thinking the grouping of actions should not be made by configuration but using a dedicated API. For example, we should be able to group also the `getItem` call only when the next item is not in the cache. 
Even if this solution work for the current use case, I'm afraid it doesn't offer us all the flexibility required for the long term goal.

Reply:
to group `getItem` call we just need to implement class for this action and add line to the test runner config.
Could you please describe how you this solution.